theforeman.org/pipelines/release/source/foreman.groovy

```diff
@@ -110,7 +84,7 @@ pipeline {
                     git url: git_url, branch: git_ref
```

Comment:
one thing I realized (but it's not a regression): `git_ref` is `develop` here, right?
So the tests can run with `develop (abcdef12)`, but the source generation could run with `develop (dedede12)` if someone pushed to `develop` in-between.

Reply:
I think I've seen some weirdness in that area before. Probably something to look into with a follow up